src/store/sagas/rootSagas.js

```diff
@@ -0,0 +1,21 @@
+import { all } from "redux-saga/effects";
+import {
+  watchAllListSaga,
+  watchAddListSaga,
+  watchEditListSaga,
+  watchDeleteListSaga
+} from "./list";
+import {
+  watchIsVisibleAddToDoInListSaga,
+  watchSetIdForAddingToDo,
+} from "./add_todo_in_list";
+export default function* rootSagas() {
+  yield all([
```

Comment:
I have better approach for this
export all sagas from 1 file as 1 and call that single watcher here only 1 time

Reply:
I have done like this
  //watcher list
  export function* watcherLists() {
    yield takeLatest(ALL_LIST_SAGA, allList);
    yield takeLatest(ADD_LIST_SAGA, addList);
    yield takeLatest(EDIT_LIST_SAGA, editList);
    yield takeLatest(DELETE_LIST_SAGA, deleteList);
  }
  export default watcherLists;

and at index.js file

import { all } from 'redux-saga/effects';
import  watcherLists  from './lists';
import watcherAddToDoInList from './add_todo_in_list';
export default function* rootSaga() {
  const sagas = [watcherLists(),watcherAddToDoInList()];
  yield all(sagas);
}